envoy/stream_info/stream_info.h

```diff
@@ -373,6 +373,16 @@ class StreamInfo {
    */
   virtual const std::string& getRouteName() const PURE;
 
+  /**
+   * @param std::string name denotes the name of the virtual cluster.
+   */
+  virtual void setVirtualClusterName(const std::string& name) PURE;
+
+  /**
+   * @return std::string& the name of the virtual cluster which got matched.
+   */
+  virtual const std::string& getVirtualClusterName() const PURE;
```

Comment:
rename this `virtualClusterName()` and describe what happens if no one calls setVirtualClusterName() first. Looking at population of this below. I think we should make this return `absl::optional<absl::string_view>` or `OptRef<std:string>` (see envoy/common/optref.h).

I see there's a pre-existing naming inconsistency within this class with getRouteName() above and. bytesReceived() below, but typically in our style guide we just name methods for what we are trying to get, without the explicit `get` verb.

Reply:
Changed to `virtualClusterName()` as suggested.